.github/workflows/updater-check.yaml

```
@@ -0,0 +1,61 @@
+name: Start Updater
+on:
+  push:
+    branches:
+    - 'yli3/**'
+  schedule:
+  - cron: '0 */3 * * *'
+
+jobs:
+  read-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.output-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Read versions
+      id: versions
+      run: |
+        VERSIONS=$(cat scanner/updater/version/RELEASE_VERSION)
+        echo "$VERSIONS" | grep -Eo '([0-9]+\.[0-9]+\.[0-9]+|[0-9]+\.[0-9]+\.x-[0-9]+-g[a-f0-9]+)' > versions.txt
+
+    - name: Generate matrix JSON
+      id: output-matrix
+      run: |
+        EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+        MATRIX=$(jq -Rs '{ versions: split("\n")[:-1] }' versions.txt)
+        echo "matrix<<EOF"$'\n'"$MATRIX"$'\n'EOF >> $GITHUB_OUTPUT
+        echo "$MATRIX"
```

Debug output still required?
```suggestion
        echo "${MATRIX}"
```